terraform/modules/private-endpoints/acr/main.tf

```diff
@@ -0,0 +1,64 @@
+locals {
+  cidr_prefix = "10.0.0.0/16"
```

Comment:
Let's move this to variables.tf instead. Perhaps this could be the same cidr prefix propagated from main.tf.

Reply:
Apologies, forgot to remove this one since I only used it for testing!